Kudu.Services/ByteRanges/HttpRequestMessageExtensions.cs

```diff
@@ -41,34 +42,48 @@ public static HttpResponseMessage CreateErrorResponse(this HttpRequestMessage re
             return rangeNotSatisfiableResponse;
         }
 
-        public static async Task<long> CopyToAsync(this HttpContent content, string filePath, ITracer tracer)
+        public static async Task<long> CopyToAsync(this HttpContent content, string filePath, ITracer tracer, bool leaveOpen = false)
```

Comment:
Consider creating a new method, and based on a feature flag switch between them.

Reply:
added new method that is only called for function app with run from package path